package.json

```diff
@@ -1,11 +1,13 @@
 {
-  "name": "d2l-typography",
   "description": "Polymer and Sass custom styles/mixins for applying D2L typography",
-  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/typography.git"
+  },
+  "name": "d2l-typography",
   "scripts": {
     "build:sass": "node-sass --output-style expanded ./test/acceptance/typography.css.scss ./test/acceptance/typography.css",
-    "postinstall": "polymer install --variants",
-    "lint": "polymer lint -i *.html",
+    "lint": "npm run lint:wc && npm run lint:js",
```

Comment:
`lint:wc` doesn't exist

Reply:
ya.  do we want to clean this up (make consistent) as we go?  alternative looks like the script would need to be a little more flexible.

Reply:
My original thinking was that we'd take this opportunity to make things consistent (make the `package.json`s all use the same script conventions before running the script)

Reply:
Sounds good / agree.